src/NuGet.Core/NuGet.Common/Telemetry/TelemetryEvent.cs

```diff
@@ -63,5 +65,16 @@ public void AddPiiData(string key, object value)
         {
             return _piiProperties;
         }
+
+
+        public void AddListOfPiiValues(string key, IEnumerable<string> listOfValues)
```

Comment:
@zivkan I see your changes are a bit more generic? 
You think we should rename this to AddComplexPiiValues?

Reply:
I think it's ok as-is. When I rebase my work and deal with the merge conflicts, I'll rename it while making it more generic.

Reply:
Makes sense. 
Resolving this.